Libraries/Lists/VirtualizedList.js

```diff
@@ -960,7 +960,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     } = this.props;
     const {data, horizontal} = this.props;
     const isVirtualizationDisabled = this._isVirtualizationDisabled();
-    const inversionStyle = this.props.inverted
+    const inversionStyle = this.props.inverted && Platform.OS !== 'macos' // TODO(macOS GH#774)
```

Comment:
Can we leave a comment above this line saying we support inversion natively in macOS ScrollView and thus don't need the "Style hack"?

Reply:
Sure! Do you want to provide the specific wording I should choose? ;-)